tests/test_schemas.py

```diff
@@ -61,3 +71,48 @@ def test_deployment_schema(status):
             },
         },
     })
+
+
+@pytest.mark.parametrize(
+    'state',
+    ConfigurationStateChoices,
+)
+def test_configuration_schema(state, file):
+    now = datetime.utcnow()
+    iso = now.isoformat()
+    serializer = ConfigurationSchema(
+        id='CFL-000-000-000',
+        file=FileSchema(
+            id=file.id,
+            name=file.name,
+            location=file.location,
+            size=file.size,
+        ),
+        deployment={'id': 'DPL-000-000-000'},
+        state=state,
+        events={
+            'created': {'at': now, 'by': 'SU-295-689-628'},
+            'updated': {'at': now, 'by': 'SU-295-689-628'}},
```

Comment:
[COD-C] Propose to move closing bracket to the next line to improve readability. Ok for you?

Reply:
Agree, lets change that!